phabfive/maniphest.py

```diff
@@ -293,3 +374,18 @@ def recurse_commit_transactions(task_config, parent_task_config):
 
         # Always start with a blank parent
         recurse_commit_transactions(parsed_root_data, None)
+
+def days_to_unix(days):
+    """
+    Convert days into a UNIX timestamp.
+    """
+    if days:
```

Comment:
This if-logic should be moved out to the calling function. This is also a utility function, so apply the same thinking as `format_timestamp()`.